jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull-e2e.yaml

```diff
@@ -181,8 +181,8 @@
                 export E2E_TEST="true"
                 export E2E_DOWN="true"
                 export E2E_OPT="--check_version_skew=false"
-                # Force to use container-vm.
-                export KUBE_GKE_IMAGE_TYPE="debian"
+                # ContainerVM is not supported any more. Use GCI.
+                export KUBE_GKE_IMAGE_TYPE="gci"
```

Comment:
I think the reason this wasn't working is because the image is called "debian" when on GCE but it's called "container_vm" when it's on GKE (crazy eh?). We should keep "container_vm" around a little while longer (maybe a week) so we have a comparison point vs "gci". I just sent #665 for this. Once the `e2e-gke` job has it set to `gci`, it's no different than `gci-e2e-gke` job and it won't make sense to keep both around.